test/acceptance/keywords/screenshots.robot

```diff
@@ -79,3 +79,72 @@ Capture page screenshot with escaped braces
     Should Be Equal    ${file}    ${OUTPUTDIR}${/}screenshot-and-index${/}brackets-{index}-name.png
     ${file} =    Capture Page Screenshot    ${OUTPUTDIR}${/}screenshot-and-index${/}brackets-{{index-name.png
     File Should Exist    ${OUTPUTDIR}${/}screenshot-and-index${/}brackets-{index-name.png
+
+Capture page screenshot computed name is unique
+    [Documentation]  verify that computed filenames are unique
+    [Tags]     issue-502
```

Comment:
We do not use tags in this context, although it is good idea. Could you remove the tags.